bleep-model/src/scala/bleep/rewrites/keepSelectedProjects.scala

```diff
@@ -1,10 +1,10 @@
 package bleep
 package rewrites
 
-case class keepSelectedProjects(selectedProjectGlobs: List[String]) extends BuildRewrite {
+case class keepSelectedProjects(selectedProjectGlobs: Array[model.ProjectGlob]) extends BuildRewrite {
```

Comment:
Why the changes to `Array`? If `List` is not good for some reason, why not at least `ArraySeq`?